src/core/mod.rs

```diff
@@ -305,6 +310,30 @@ impl BPF {
         Ok(())
     }
 
+    #[cfg(any(
```

Comment:
This might just be me being a rust noob, but the logic here is hard for me to follow. Am I correct in assuming that [lines 323 - 328](https://github.com/rust-bpf/rust-bcc/pull/68/files#diff-ad369e8c88b22ab972c1c24ccad4d22fR323-R328) aren't necessary? It seems to me like those feature strings being listed in the outer `any` means their value would never matter within the `not(any(`. If this is true then there are other sections of logic for which the same feedback applies.

Too bad we can't do something like
```
#[cfg(crate_version("bcc-sys") >= 0.6)
```

or
```
#[cfg(accessible(bcc_sys::bccapi::bpf_attach_raw_tracepoint))]
```

or
```
#[cfg(feature("bcc-sys", bpf_attach_raw_tracepoint))
```

Reply:
Yes, that's too bad, I used `not(feature = "specific")` instead of `not(any(`